.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+# Dependabot dependency version checks / updates
+
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
let's get the github-action dependabot check in there as well (like for binstall), that was useful

Reply:
Done!